static/app/views/insights/mobile/ui/settings.ts

```diff
@@ -3,8 +3,6 @@ import {t} from 'sentry/locale';
 export const MODULE_TITLE = t('Mobile UI');
 export const BASE_URL = 'ui';
 
-export const MODULE_DESCRIPTION = t("Improve your application's responsiveness.");
```

Comment:
note: I think all of those `settings.ts` files have the same structure, but `MODULE_DESCRIPTION` is often not used.

Unless there is something I’m missing and this gets dynamically required somehow?